generator/internal/mock_connection_generator.cc

```diff
@@ -94,7 +94,7 @@ class $mock_connection_class_name$ : public $product_namespace$::$connection_cla
                  {IsResponseTypeEmpty,
                   // clang-format off
     "\n  MOCK_METHOD(Status,\n",
-    "\n  MOCK_METHOD(StatusOr<$response_type$>,\n"},
```

Comment:
Seems like the only case where the return type *might* contain a comma is already handled.  Are you sure you want to add this code if it will have no effect?